framework/include/auxkernels/AuxKernel.h

```diff
@@ -48,6 +49,9 @@ InputParameters validParams<AuxKernel>();
 template <>
 InputParameters validParams<VectorAuxKernel>();
 
+template <>
+InputParameters validParams<ArrayAuxKernel>();
```

Comment:
Legacy code. Can remove

Reply:
We actually have `validParams<VectorAuxKernel>` and `validParams<AuxKernel>` right above this. So it is better to keep this and remove all three together in the future.